protocols/migration/migration_reports_5_0_0_to_reports_6_0_0.py

```diff
@@ -25,3 +25,172 @@ def migrate_interpretation_request_rd(self, old_instance):
+        if old_frequencies is None:
+            return None
+        return [self.migrate_allele_frequency(old_frequency=old_frequency) for old_frequency in old_frequencies]
+
+    def migrate_allele_frequency(self, old_frequency):
+        new_frequency = self.convert_class(self.new_model.AlleleFrequency, old_frequency)
+        return self.validate_object(object_to_validate=new_frequency, object_type=self.new_model.AlleleFrequency)
+
+    def migrate_allele_origins(self, old_origins):
+        return [self.migrate_allele_origin(origin=origin) for origin in old_origins]
+
+    def migrate_allele_origin(self, origin):
+        origin_map = {
+            self.old_model.AlleleOrigin.de_novo_variant: self.new_model.AlleleOrigin.de_novo_variant,
+            self.old_model.AlleleOrigin.germline_variant: self.new_model.AlleleOrigin.germline_variant,
+            self.old_model.AlleleOrigin.maternal_variant: self.new_model.AlleleOrigin.maternal_variant,
+            self.old_model.AlleleOrigin.paternal_variant: self.new_model.AlleleOrigin.paternal_variant,
+            self.old_model.AlleleOrigin.pedigree_specific_variant: self.new_model.AlleleOrigin.pedigree_specific_variant,
+            self.old_model.AlleleOrigin.population_specific_variant: self.new_model.AlleleOrigin.population_specific_variant,
+            self.old_model.AlleleOrigin.somatic_variant: self.new_model.AlleleOrigin.somatic_variant,
+        }
+        return origin_map.get(origin)
+
+    def migrate_variant_calls(self, variant_calls):
+        return [self.migrate_variant_call(variant_call=variant_call) for variant_call in variant_calls]
+
+    def migrate_variant_call(self, variant_call):
+        new_variant_call = self.convert_class(self.new_model.VariantCall, variant_call)
+
+        return self.validate_object(
+            object_to_validate=new_variant_call, object_type=self.new_model.VariantCall
+        )
+
+    def migrate_report_events(self, report_events):
+        return [self.migrate_report_event(report_event=report_event) for report_event in report_events]
+
+    def migrate_report_event(self, report_event):
+        new_report_event = self.convert_class(self.new_model.ReportEvent, report_event)
+        new_report_event.phenotypes = self.migrate_phenotypes(phenotypes=report_event.phenotypes)
+        new_report_event.variantConsequences = self.migrate_variant_conseuqences(variant_consequences=report_event.variantConsequences)
+        new_report_event.genePanel = self.migrate_gene_panel(gene_panel=report_event.genePanel)
+        new_report_event.genomicEntities = self.migrate_genomic_entities(genomic_entities=report_event.genomicEntities)
+        new_report_event.variantClassification = self.migrate_variant_classification(classification=report_event.variantClassification)
+        return self.validate_object(object_to_validate=new_report_event, object_type=self.new_model.ReportEvent)
+
+    def migrate_gene_panel(self, gene_panel):
```

Comment:
why do we need to map these fields? I thought these would be filled by `self.convert_class()` as they are named the same.